raiden/ui/console.py

```diff
@@ -234,51 +232,6 @@ def _run(self):  # pylint: disable=method-hidden
         print('Tip:' + OKBLUE)
         print_usage()
 
-        # Remove handlers that log to stderr
-        root = structlog.get_logger()
-        for handler in root.handlers[:]:
-            if isinstance(handler, StreamHandler) and handler.stream == sys.stderr:
-                root.removeHandler(handler)
-
-        stream = io.StringIO()
-        handler = StreamHandler(stream=stream)
-        handler.formatter = Formatter(u'%(levelname)s:%(name)s %(message)s')
-        root.addHandler(handler)
-
-        def lastlog(n=10, prefix=None, level=None):
-            """ Print the last `n` log lines to stdout.
-            Use `prefix='p2p'` to filter for a specific logger.
-            Use `level=INFO` to filter for a specific level.
-            Level- and prefix-filtering are applied before tailing the log.
-            """
-            lines = (stream.getvalue().strip().split('\n') or [])
-            if prefix:
-                lines = [
-                    line
-                    for line in lines
-                    if line.split(':')[1].startswith(prefix)
-                ]
-            if level:
-                lines = [
-                    line
-                    for line in lines
-                    if line.split(':')[0] == level
-                ]
-            for line in lines[-n:]:
-                print(line)
-
-        self.console_locals['lastlog'] = lastlog
-
-        err = io.StringIO()
-        sys.stderr = err
-
-        def lasterr(n=1):
-            """ Print the last `n` entries of stderr to stdout. """
-            for line in (err.getvalue().strip().split('\n') or [])[-n:]:
-                print(line)
-
-        self.console_locals['lasterr'] = lasterr
-
```

Comment:
Removing this will cause the console to be spammed by log lines and thereby be almost unusable. Is that intentional?